### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tiserett**
- [DEMO LINK](https://tiserett.github.io/layout_catalog/)
- [TEST REPORT LINK](https://tiserett.github.io/layout_catalog/report/html_report/)

I can't put stars.scss file into 'blocks' folder because of problems which I recently described in thread in slack.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,328 @@
...
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <section class="container">
+        <div class="card container__card" data-qa="card">
```
Comment (kir8mir) [inline src/index.html:62]:
```suggestion
        <div class="card" data-qa="card">
```
I don't actually understand for what you are using mix here. There are no one style for this class.

Comment (tiserett) [reply src/index.html:62]: I actually used it in cards block

Comment (kir8mir) [reply src/index.html:62]: It's ok. I'm speaking about you did't put any style code in this class. For what you need it?

Comment (tiserett) [reply src/index.html:62]: check .container in cards.scss again

Comment (kir8mir) [reply src/index.html:62]: I see now, take my apologies! ;)

**Review by kir8mir — APPROVED** [review]
Can't to find any inappropriate code. Nice job!

File: src/styles/blocks/header.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
```
Comment (AlinaMatuschak) [inline src/styles/blocks/header.scss:13]: Should create separate file for nav

File: src/styles/stars.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.stars {
```
Comment (AlinaMatuschak) [inline src/styles/stars.scss:1]: there should work stars--4 class too

**Comment by tiserett** [comment]
putted nav in it's own file, fixed problems with container, didn't really understood what you meant by telling me to calc width, so I missed this part, as well as part with stars, I guess these two parts aren't essential, correct me if I'm mistaken

File: src/styles/blocks/cards.scss [hunk 4]
```diff
@@ -0,0 +1,103 @@
+.content {
```
Comment (AlinaMatuschak) [inline src/styles/blocks/cards.scss:1]: Separate file with name content

File: src/styles/blocks/cards.scss [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  margin: 0 auto;
+  width: 1024px;
+}
+
+.container {
```
Comment (AlinaMatuschak) [inline src/styles/blocks/cards.scss:6]: Separate file with name container

File: src/styles/blocks/cards.scss [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
+  &__card:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+}
+
+.card {
```
Comment (AlinaMatuschak) [inline src/styles/blocks/cards.scss:22]: Rename this file to card (this file will responsible for card styles)

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
- still don’t work all stars classes

Screenshot 2022-07-19 at 12 24 49

File: src/styles/stars.scss [hunk 7]
```diff
@@ -7,7 +7,12 @@
     margin-right: 4px;
     background: url(../images/star-active.svg) center no-repeat;
 
-    &:last-of-type {
+    .stars--0 &:nth-of-type(n+1),
+    .stars--1 &:nth-of-type(n+2),
+    .stars--2 &:nth-of-type(n+3),
+    .stars--3 &:nth-of-type(n+4),
+    .stars--4 &:nth-of-type(n+5),
+    .stars--5 &:nth-of-type(n+6) {
```
Comment (AlinaMatuschak) [inline src/styles/stars.scss:15]: Will be nice to use [cycle](https://sass-lang.com/documentation/at-rules/control/for)

File: src/styles/blocks/container.scss [hunk 8]
```diff
@@ -3,7 +3,9 @@
   flex-wrap: wrap;
   justify-content: center;
 
-  padding: 50px 40px;
+  padding: 50px 0;
+  margin: 0 auto;
+  width: calc(200px * 4 + 48px * 3);
```
Comment (AlinaMatuschak) [inline src/styles/blocks/container.scss:8]: use variables for 200px and 48px

**Review by AlinaMatuschak — APPROVED** [review]

**Pull request closed without merging**